packages/react-native/Libraries/Components/Touchable/TouchableNativeFeedback.js

```diff
@@ -96,6 +95,13 @@ type State = $ReadOnly<{
 }>;
 
 class TouchableNativeFeedback extends React.Component<Props, State> {
+  hostRef: {current: React.ElementRef<typeof View> | null, ...};
```

Comment:
This approach does not allow you to pass a ref from a parent via prop. Could you make changes to allow a ref to forward and merge these refs together? `TouchableOpacity` has a small example. Also, could you make these changes with `TouchableNativeFeedback.macos` as well?